ui/push-health/Job.jsx

```diff
@@ -44,6 +45,9 @@ class Job extends PureComponent {
                   color="lightgray"
                 />
               )}
+              {!!failedInParent && (
+                <Badge color="warning">Failed in parent</Badge>
```

Comment:
I show this as a yellow `warning`, but I almost wonder if it would be more appropriate to have it be a green `success`.  Since the same error existing in the parent really means they likely don't have to worry about it.

Reply:
What about an "info" color?  It's just something to take note of?

Reply:
Sure, that works.  I'll go with info.